pkgs/by-name/de/decent-sampler/package.nix

```
@@ -46,6 +71,11 @@ buildFHSEnv {
 
   runScript = "decent-sampler";
 
+  extraInstallCommands = ''
+    cp -r ${decent-sampler.outPath}/lib $out/lib
+    cp -r ${decent-sampler.outPath}/share $out/share
```

```suggestion
    cp -r ${decent-sampler}/lib $out/lib
    cp -r ${decent-sampler}/share $out/share
```
that should be the same

---

Done.